### PR TITLE
new docs for lab equipment

### PR DESCRIPTION
**Pull request by mgheorghe**
- removed the need for password
- config file for mgmt server
  - dhcp config
  - nftables config
- updates to arista switch config file

### CODE REVIEW

**Comment by mgheorghe** [comment]
1 test passes 2 fail.
login works

================================================================== short test summary info ==================================================================
FAILED demos/tgen/test_tgen_demo.py::test_server_to_dpuhost_via_dpu - Exception: code: 500
FAILED demos/tgen/test_tgen_demo.py::test_dpuhost_to_dpuhost_via_dpu - requests.exceptions.ConnectionError: HTTPSConnectionPool(host='172.22.1.2', port=8443): Max retries exceeded with url: /config (Caused by NewConnectionE...
=============================================================== 2 failed, 1 passed in 28.58s ================================================================
opi@tgen1:~/opi-poc$

File: demos/tgen/test_tgen_demo.py [hunk 1]
```diff
@@ -3,26 +3,12 @@
...
-
-
-
+import requests
 
 from testbed import *
 
-
-
+HOME_FOLDER = r'/home/opi/actions-runner/_work/opi-poc/opi-poc'
```
Comment (glimchb) [inline demos/tgen/test_tgen_demo.py:10]: I don't like globals

**Review by glimchb — APPROVED** [review]

File: demos/tgen/test_tgen_demo.py (hunk 1 above)
Comment (mgheorghe) [reply demos/tgen/test_tgen_demo.py:10]: i need to play with the action runner to see what is the relative path, and make it work when ran via git clone manually and via action runner, it's on TODO list

**Comment by glimchb** [comment]
DCO  fails, please use `-s` flag in the `git commit -s`

**Comment by mgheorghe** [comment]
created many smaller  PRs 
- https://github.com/opiproject/opi-poc/pull/958
- https://github.com/opiproject/opi-poc/pull/959
- https://github.com/opiproject/opi-poc/pull/960
- https://github.com/opiproject/opi-poc/pull/961
- https://github.com/opiproject/opi-poc/pull/962
- https://github.com/opiproject/opi-poc/pull/963
- https://github.com/opiproject/opi-poc/pull/964

**Pull request closed without merging**